### PR TITLE
Feat/challenge 03 loops conditionals

### PR DESCRIPTION
**Pull request by JosepRubio**

### CODE REVIEW

File: challenge_03_loops-conditionals/README.md
Comment (tictools) [inline challenge_03_loops-conditionals/README.md:1]: ⚠️ FIX: move file at project root level

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    4. If it is equal to "cat", print out "Meow!" to the screen
+    5. If it is equal to "dog", print out "Woof!" to the screen
+    6. If it is neither "cat" nor "dog", print out "Must be an alien"
+    7. At the end of the program print out: "--- Animal check complete ---"
+    8. Change animal to different values to make sure it works in all cases
+*/
+
+console.log("--- animal check program ---");
+let animal = "cat";
+if (animal == "cat") {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js]: ⚠️ FIX: Change animal to different values to make sure it works in all cases

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    Create a program that starts with a number variable.
+    The program will return the variable is even or odd.
+
+    Change the variable to test it works for different cases.
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+
+let number = 5;
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js]: ⚠️ FIX: Change the variable to test it works for different cases. Print result.

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+for (let i = 10; i < 40; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  if (i % 2 === 0) {
+    console.log(i + " is even");
+  }
+}
+
+for (let j = 10; j < 40; j++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js]: You should take in account what `j` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+for (let i = 1; i <= 100; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable


This is the output of your script
```
Checkpoint!10
Checkpoint!20
Checkpoint!30
Checkpoint!40
Half way there!50
Checkpoint!60
Checkpoint!70
Checkpoint!80
Checkpoint!90
You made it!100
All, done!
```

have you tested it before pushing? 🤔

⚠️ FIX: needs work.

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    2. Create a while loop that will print out all the odd numbers between 10 and 40.
+*/
+
+let number = 10;
+while (number <= 40) {
+  if (number % 2 === 0) console.log("even: ", number);
+  else if (number % 2 === 1) console.log("odd: ", number);
+  number++;
+}
+/*let number = 10;
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js]: ⚠️ FIX: remove commented code

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
+/*
+    1. Create a while loop that will print out all the even numbers between 10 and 40.
+
+    2. Create a while loop that will print out all the odd numbers between 10 and 40.
+*/
+
+let number = 10;
+while (number <= 40) {
+  if (number % 2 === 0) console.log("even: ", number);
+  else if (number % 2 === 1) console.log("odd: ", number);
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js]:
```suggestion
  else if (number % 2 !== 0) console.log("odd: ", number);
```

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_02.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+let number = 0;
+while (number <= 100) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_02.js]: This is the output of your script
```
Checkpoint!10
Checkpoint!20
Checkpoint!30
Checkpoint!40
Half way there!50
Checkpoint!60
Checkpoint!70
Checkpoint!80
Checkpoint!90
You made it!100
All, done!
```

have you tested it before pushing? 🤔

⚠️ FIX: needs work.

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by tictools — APPROVED** [review]

**Review by UnPiquito — APPROVED** [review]

**Pull request merged**